examples/hackweek_demo/run_demo.py

```diff
@@ -70,8 +67,15 @@ def get_ticker(data):
 
 
 while True:
+    pool_config_data = postgres.get_pool_config(session)
     txn_data = postgres.get_transactions(session, start_block=start_block)
     pool_info_data = postgres.get_pool_info(session, start_block=start_block)
+    checkpoint_info = postgres.get_checkpoint_info(session, start_block=start_block)
+
+    assert max(pool_info_data.index) == max(checkpoint_info.index), (
```

Comment:
I think we should handle this case a bit more gracefully. I'm assuming you're worried about a misalignment of pool info data and checkpoint info data since acquire_data.py writes these fields sequentially. I think that if this is the case, lets just throw out any information that doesn't exist in both tables. This way, the streamlit doesn't crash due to this assertion, and instead can continue on.

Reply:
I was thinking of implementing a more complicated approach with re-querying, but after discussing with you, let's just go with the simple "throw out the data" approach, since we're running non-stop queries in Streamlit, increasing update speed is actually a nice benefit. let me see if I can do that quick.

Reply:
implemented in c5e85d1580c60fb20190b2f9861fac14fc946641